### PR TITLE
Add terminator setup

### PR DESCRIPTION
**Pull request by fliiiix**
Don't @ me but thats what im using

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +47,24 @@ to include the theme located in this repository at `./kitty/edelweiss.conf`.
...
+and `palette` in your default profile.
+
+```
+[profiles]
+  [[edelweiss]]
+    background_color = "#ffffff"
+    foreground_color = "#333333"
+    show_titlebar = False
+    scrollback_lines = 10000
+    palette = "#000000:#dc2626:#16a34a:#ca8a04:#2563eb:#db2777:#0891b2:#dddddd:#767676:#f87171:#4ade80:#facc15:#60a5fa:#f472b6:#22d3ee:#ffffff"
```
Comment (timofurrer) [inline README.md:60]: Can you post a screenshot of this in this MR? I'm not using terminator, but happy to include it. 

Also, the colors in the other themes are from the Tailwind CSS palette, see [here](https://tailwindcss.com/docs/customizing-colors).

And how they are used in the [neovim color schemes](https://github.com/timofurrer/weiss/blob/1cb489179a0c0ebae28e4796e772c08e09e3c072/nvim/lua/edelweiss/colors.lua#L3).

What do you think about using the `600` shade of each base color in this palette? And black as `#333333` - I don't like `#000000` too much, but no show stopper if you really want "black".

File: README.md [hunk 2]
```diff
@@ -46,6 +47,24 @@ to include the theme located in this repository at `./kitty/edelweiss.conf`.
...
+Add a new profile to `.config/terminator/config` or change the `background_color`, `foreground_color`
+and `palette` in your default profile.
+
+```
+[profiles]
+  [[edelweiss]]
+    background_color = "#ffffff"
+    foreground_color = "#333333"
+    show_titlebar = False
+    scrollback_lines = 10000
```
Comment (timofurrer) [inline README.md]: I suggest to either remove the unrelated config fields or provide a config file. Are the includes in terminator for themes?

File: README.md (hunk 1 above)
Comment (fliiiix) [reply README.md:60]: i tried to copy all the colors from kitty so it's just color0-color15

```
#: black (non-tailwind)
color0 #000000
color8 #767676
```

so i would just keep it consistent up to you if you want to change 
i personally didn't see a problem yet with full black


and it looks something like this for me

image

File: README.md (hunk 2 above)
Comment (fliiiix) [reply README.md]: well :sweat_smile: it's a bit 'interesting' there is no thing like a theme or something you can install it's all part of a single config file this one -> 

```
$ cat .config/terminator/config 
[global_config]
  inactive_color_offset = 0.809989142237
[keybindings]
[profiles]
  [[default]]
    background_color = "#ffffff"
    foreground_color = "#333333"
    show_titlebar = False
    scrollback_lines = 10000
    palette = "#000000:#dc2626:#16a34a:#ca8a04:#2563eb:#db2777:#0891b2:#dddddd:#767676:#f87171:#4ade80:#facc15:#60a5fa:#f472b6:#22d3ee:#ffffff"
    login_shell = True
    bold_is_bright = True
[layouts]
  [[default]]
    [[[child1]]]
      parent = window0
      type = Terminal
    [[[window0]]]
      parent = ""
      type = Window
[plugins]
```

but im down to remove `show_titlebar` , `scrollback_lines` and `login_shell` as they are not related to the color config if you want me to

File: README.md (hunk 1 above)
Comment (timofurrer) [reply README.md:60]: Okay, so these are the proper colors then I suppose ...

File: README.md [hunk 3]
```diff
@@ -46,6 +47,21 @@ to include the theme located in this repository at `./kitty/edelweiss.conf`.
 echo 'include ~/.config/edelweiss/kitty/edelweiss.conf' >> ~/.config/kitty/kitty.conf
 ```
 
+### terminator
+
+Add a new profile to `.config/terminator/config` or change the `background_color`, `foreground_color`
+and `palette` in your default profile.
```
Comment (timofurrer) [inline README.md:53]: What about `bold_is_bright` that is still set below? I suggest either mentioning it here, too or remove it from the snippet? WDYT?

**Comment by fliiiix** [comment]
pls merge or close :)

**Review by timofurrer — COMMENTED** [review]
@fliiiix had a pending comment actually

**Comment by timofurrer** [comment]
@fliiiix please rebase, soo :)

File: README.md (hunk 3 above)
Comment (fliiiix) [reply README.md:53]: idk tbh i just removed it and it looks ok to me

**Review by timofurrer — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ edelweiss, `/ˈeɪdəlvaɪs/`. A de**light**ful color scheme for my personal ter
 edelweiss currently supports:
 
 - [kitty](https://sw.kovidgoyal.net/kitty/) - *The fast, feature-rich, CPU based terminal emulator.*
+- [terminator](https://github.com/gnome-terminator/terminator) - *Multiple GNOME terminals in one window .*
```
Comment (timofurrer) [inline README.md]:
```suggestion
- [terminator](https://github.com/gnome-terminator/terminator) - *Multiple GNOME terminals in one window.*
```

**Review by timofurrer — CHANGES_REQUESTED** [review]
Soooorrry, this needs to be fixed ^^

**Review by timofurrer — APPROVED** [review]

**Pull request merged**